hw-2/dirtree.js

```diff
@@ -0,0 +1,110 @@
+/* Usage: node dirtree [dist] [-d/--depth?: number] */
+
+const args = process.argv.slice(2);
+
+if (args.length === 0) {
+	throw new Error('Please pass a file/directory as an argument');
+}
+
+let maxDepth = Infinity;
+let dist = undefined;
+
+for (let i = 0; i < args.length; ) {
```

Comment:
wow, covering arguments seem to involve already some code logic 👍 we'll check some tools to make it easier on Friday